### DISCUSSION TITLE
Sengled E1C-NB7 not supported but listed

### DISCUSSION THREAD

**Discussion by wshankles** [Koenkk/zigbee2mqtt#11563] [Devices]
I paired my sengled E1C-NB7 plugs and the logs say it is not supported. The zigbee2mqtt docs say otherwise.

**Comment by benley** [comment]
FWIW, I have a few of the exact same model of plugs paired with zigbee2mqtt and they're both marked as supported and working fine.

**Reply by deviantintegral** [reply]
Yes, I have 6 myself with no issues. Are you running an old version of zigbee2mqtt?

**Reply by Deep-Six** [reply]
Can you advise how you were able to pair these? I have one, and I am not sure if it's defective but I never get any flashing lights after holding the power toggle  button and then releasing it for between 5 and 8-15 seconds like it says to factory reset.

Thanks!

**Reply by deviantintegral** [reply]
I just paired one for my Christmas tree. There’s three colours from the LED:

- Red flashing when it’s resetting or being removed from the network.
- Purple flashing when attempting to join a network.
- Blue, on for on when it’s off and off for when it’s off!
- If you press and hold to reset but stop before the red goes on, it will join but not reset the Energy counter. If you hold longer it goes red, and resets that counter to zero.

I’ll also say that I’ve set up 10? 15? of these and I’ve never encountered a defective one. They seem to be pretty well built compared to their lightbulbs that I now avoid.

Something easy to check, did you get them in a 2 or 4 pack? I’d be surprised if they were all broken.

Any chance you accidentally got wifi plugs?

**Reply by Deep-Six** [reply]
It says zigbee right on it, and you're right, it  was a 2 pack I am not sure where the other is. I've only been able to see the blue light, never the red or purple. Do you hold the button down from the on state or the off state??

**Reply by Deep-Six** [reply]
Figured it out, strangest thing, the outlet was one of those ones that has a usb A and C in the centre, I was using the A side to plug in another device. The Sengled was maybe 3 mm from being flush to the socket. I thought, maybe it's not getting a good ground connection and sure enough. Even though the outlet was functional before I could turn the button on and off manually, I put it flush and it worked instantly to reset it. Very strange.

**Comment by PelonVela** [comment]
> I paired my sengled E1C-NB7 plugs and the logs say it is not supported. The zigbee2mqtt docs say otherwise.

I have the same problem. Did you solved it?

**Comment by AJEvans3** [comment]
I am having the same issue. No fix?

**Reply by PelonVela** [reply]
Not sure if that’s your case, but I was running and old version of zigbee2mqtt. I’m currently on version: 1.30.1-1.

I was using an old repo. Since the upgrade sengled outlets are workin correctly.

**Reply by AJEvans3** [reply]
Yeah, I’m currently running 1.30.1-1 without any success.
